ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -94,21 +84,34 @@ Menu {
         if (typeof closeHandler === "function") {
             closeHandler()
         }
+        d.maxDelegateImplWidth = 0
     }
 
     QtObject {
         id: d
         //helper property to get the max implicit width of the delegate
         property real maxDelegateImplWidth: 0
+
+        function calcMaxWidth() {
+            for (let i = 0; i < root.count; i++) {
+                const menuItem = root.itemAt(i)
+                if (!!menuItem && menuItem.text && menuItem.enabled) {
+                    metrics.font = menuItem.font
+                    d.maxDelegateImplWidth = Math.max(d.maxDelegateImplWidth, metrics.boundingRect(menuItem.text).width +
+                                                      menuItem.indicator.width + menuItem.horizontalPadding*4 + menuItem.spacing*4)
```

Comment:
I'm just curious, why `*4` and not `*2`?

Reply:
Both the menu item itself and the text have those